static/js/search_suggestion.js

```diff
@@ -731,7 +739,7 @@ exports.get_suggestions = function (base_query, query) {
 
 exports.finalize_search_result = function (result) {
     for (const sug of result) {
-        const first = sug.description.charAt(0).toUpperCase();
+        const first = sug.description.charAt(0).toLocaleUpperCase();
```

Comment:
This also feels like it should be on a commit of its own.

Reply:
I agree here.  For something like that which is a pretty clear win, I'd try to make that its own commit, even if I had to rebase slightly (which wouldn't be hard to manage here).

And then if anything else in the commit is problematic, we can at least fix the lack of locale-awareness.

Reply:
(Left a comment regarding this below)